utils/reflector/src/main/java/org/robolectric/util/reflector/UnsafeAccess.java

```diff
@@ -26,6 +26,8 @@ static <T> Class<?> defineClass(Class<T> iClass, String reflectorClassName, byte
 
   private static class DangerPre11 implements Danger {
     private final Unsafe unsafe;
+
+    @SuppressWarnings("RethrowReflectiveOperationExceptionAsLinkageError")
```

Comment:
Is this suppressing location correctly?

Reply:
my bad, fixed it now there is no warning about `RethrowReflectiveOperationExceptionAsLinkageError` in the logs